sky/backends/cloud_vm_ray_backend.py

```diff
@@ -62,7 +62,7 @@
 
 # Timeout (seconds) for provision progress: if in this duration no new nodes
 # are launched, abort and failover.
-_NODES_LAUNCHING_PROGRESS_TIMEOUT = 60
+_NODES_LAUNCHING_PROGRESS_TIMEOUT = 150
```

Comment:
I increase the timeout because TPU VM takes longer than 60s to be ready. But I think ideally we should directly call Cloud APIs to check the status instead of relying on `ray status`.

Reply:
Can we apply the longer timeout only for TPU VMs, not for all cases?

Reply:
Yes I just made TPU VM a special case.